parser/SqlParser.ypp

```diff
@@ -440,6 +446,9 @@ void NotSupported(const YYLTYPE *location, yyscan_t yyscanner, const std::string
   table_constraint_def  // unimplemented
   table_constraint_def_commalist        // unimplemented
   opt_table_constraint_def_commalist    // unimplemented
+  opt_column_list       // unimplemented
```

Comment:
Sounds like this will be a version of `name_commalist` that is optionally empty. You might want to just make a general-purpose `opt_name_commalist`.